<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-L45CF4AI" name="com.zmarta.ExtentReportListener.ExtentReporterNG" tests="3" failures="1" timestamp="19 Sep 2018 07:12:05 GMT" time="0.075" errors="0">
  <testcase name="demoTestFail" time="0.030" classname="com.zmarta.ExtentReportListener.ExtentReporterNG"/>
  <testcase name="demoTestPass" time="0.037" classname="com.zmarta.ExtentReportListener.ExtentReporterNG"/>
  <testcase name="demoTestSkip" time="0.008" classname="com.zmarta.ExtentReportListener.ExtentReporterNG">
    <failure type="org.testng.SkipException" message="This test case not ready for execution">
      <![CDATA[org.testng.SkipException: This test case not ready for execution
	at com.zmarta.ExtentReportListener.ExtentReporterNG.demoTestSkip(ExtentReporterNG.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- demoTestSkip -->
</testsuite> <!-- com.zmarta.ExtentReportListener.ExtentReporterNG -->
